server/monitor.go

```diff
@@ -337,3 +341,19 @@ func updateUsage(v *Varz) {
 	v.CPU = pcpu
 	v.Cores = numCores
 }
+
+// ResponseHandler handles responses for monitoring routes
+func ResponseHandler(w http.ResponseWriter, r *http.Request, data []byte) {
+	// Get callback from request
+	callback := r.FormValue("callback")
```

Comment:
probably better off just using `r.URL.Query().Get("callback")` as you only need to check the URL query params